src/components/dashboard/components/concepts/ConceptList.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Card from './ConceptCard';
+import Loader from '../../../Loader';
+
+const ConceptList = ({ concepts, fetching }) => {
+  if (concepts.length >= 1) {
+    return (
+      <table className="table table-bordered table-striped">
+        <thead className="header text-white">
+          <tr>
+            <th scope="col"><div style={{ maxWidth: '15rem' }}>Name</div></th>
+            <th scope="col">Class</th>
+            <th scope="col">Datatype</th>
+            <th scope="col">Source</th>
+            <th scope="col">ID</th>
+          </tr>
+
+        </thead>
+        <tbody>
+
+          {concepts.map(concept => (
+            <Card concept={concept} key={concept.uuid} />
```

Comment:
Could you change the component name from card to table to avoid confusion

Reply:
Right